cluster/raft_apply_endpoints.go

```diff
@@ -152,6 +152,27 @@ func (s *Raft) DeleteReplicaFromShard(ctx context.Context, class, shard, targetN
 	return s.Execute(ctx, command)
 }
 
+func (s *Raft) SyncShard(ctx context.Context, collection, shard, nodeId string) (uint64, error) {
+	if collection == "" || shard == "" || nodeId == "" {
+		return 0, fmt.Errorf("empty class or shard or sourceNode or targetNode : %w", schema.ErrBadRequest)
```

Comment:
nit: there should not be space between `targetNode` and `:` - `or targetNode :` -> `or targetNode:`